src/marks/mark.py

```diff
@@ -21,13 +22,16 @@
 }
 
 hat_shapes = {
-    "splat": "fourPointStar",
-    "fox": "chevron",
-    "-wing": "threePointStar",
-    "-hole": "hole",
-    "-frame": "frame",
-    "-curve": "curve",
-    "-stare": "eye",
+    "ex": "ex",
```

Comment:
As discussed, we leave them all on by default, but they get filtered if they're not active